src/MICore/Debugger.cs

```diff
@@ -1061,6 +1062,27 @@ private void OnOutOfBand(string cmd)
                 string status = _miResults.ParseCString(cmd.Substring(8));
                 OnStateChanged("stopped", status);
             }
+            else if (cmd.StartsWith("stopped", StringComparison.Ordinal))
+            {
+                if (PlatformUtilities.IsWindows() &&
+                    this.LaunchOptions is LocalLaunchOptions &&
+                    ((LocalLaunchOptions)this.LaunchOptions).ProcessId != 0 &&
+                    this.MICommandFactory.Mode == MIMode.Gdb &&
+                    !this.IsCygwin
+                    )
```

Comment:
could this get hit in the MDD scenarios? in that case we using LocalLaunchOptions on windows.